podman_compose.py

```diff
@@ -772,6 +772,27 @@ def container_to_cpu_res_args(cnt, podman_args):
             str(mem_res).lower(),
         ))
 
+    # Handle pids limit from both container level and deploy section
+    pids_limit = cnt.get("pids_limit")
+    deploy_pids = None
```

Comment:
`limits.get("pids")` would be equivalent, no?

Reply:
Yes, definitely